CONTRIBUTING.rst

```diff
@@ -66,8 +66,8 @@ Ready to contribute? Here's how to set up `proxysql_tools` for local development
 
 3. Install your local copy into a virtualenv::
 
-    $ make virtualenv
-    $ source env/bin/activate
+    $ mkvirtualenv proxysql-tools
```

Comment:
For this do we expect the user to first install virtualenvwrapper package by hand? Why do we need to change this?